articles/service-fabric/service-fabric-understand-and-troubleshoot-with-system-health-reports.md

```diff
@@ -474,12 +474,18 @@ Visual Studio 2015 diagnostic events: RunAsync failure in **fabric:/HelloWorldSt
 
 [1]: ./media/service-fabric-understand-and-troubleshoot-with-system-health-reports/servicefabric-health-vs-runasync-exception.png
 
+### Replicator system health reports
+**Replication queue full:**
+**System.Replicator** reports a warning when the replication queue is full. On the primary, the replication queue usually becomes full because one or more secondary replicas are slow to acknowledge operations. On the secondary, this usually happens when the service is slow to apply the operations. The warning is cleared when the queue is no longer full.
 
-### Replication queue full
-**System.Replicator** reports a warning if the replication queue is full. On the primary, this usually happens because one or more secondary replicas are slow to acknowledge operations. On the secondary, this usually happens when the service is slow to apply the operations. The warning is cleared when the queue is no longer full.
+* **SourceId**: System.Replicator
+* **Property**: **PrimaryReplicationQueueStatus** or **SecondaryReplicationQueueStatus**, depending on the replica role.
```

Comment:
Most entries also have **Next steps**, which tell the user what to do next. Is there some action needed? If so, describe possible next steps. Otherwise, say that it's usually self mitigated by the system.

Reply:
added the next steps.

Reply:
added the next steps